app/views/layouts/base.html.erb

```diff
@@ -109,6 +110,7 @@
         false,
       ) %>
   <%= javascript_packs_tag_once('application', prepend: true) %>
+  <%= javascript_packs_tag_once('track-errors') if BrowserSupport.supported?(request.user_agent) %>
```

Comment:
Question: Can we defer this?

Reply:
> Question: Can we defer this?

I think we can and should make this `<script async defer>`, with the benefit being that the page can finish being considered "loaded" without being blocked by the tracker.

I still need to get some clarification about `async` script load order when combined with non-async scripts ([related Slack discussion](https://gsa-tts.slack.com/archives/C032KSPPQ/p1693405267422059)), and this'll require some revisions to `ScriptHelper` / `AssetSources` to make sure the extra attributes are carried across to the rendered script tag.

I'll plan to explore this as a follow-on task.